fiscal_epos_print/models/account.py

```diff
@@ -15,5 +15,6 @@ class AccountTax(models.Model):
 
     @api.constrains('fpdeptax')
     def _validate_fpdeptax(self):
-        if not re.search(regex, self.fpdeptax):
-            raise ValidationError("Department ID number range [1 - 99]")
+        for tax in self:
+            if not re.search(regex, tax.fpdeptax):
+                raise ValidationError("Department ID number range [1 - 99]")
```

Comment:
```suggestion
                raise ValidationError(_("Department ID number range [1 - 99]"))
```

Reply:
Questo andrebbe comunque in un commit separato.
Per tracciatura più agevole e per non bloccare questa PR, lo metterei in un'altra PR (@GSLabIt hai mica voglia di farla?).

Reply:
K